backend/manager/modules/bll/src/main/java/org/ovirt/engine/core/bll/storage/disk/UpdateDiskCommand.java

```diff
@@ -786,7 +788,12 @@ public String getDiskAlias() {
     public Map<String, String> getJobMessageProperties() {
         if (jobProperties == null) {
             jobProperties = super.getJobMessageProperties();
-            jobProperties.put("diskalias", getDiskAlias());
+            List<String> vmNames = vmsDiskOrSnapshotAttachedTo
+                    .stream()
+                    .map(VM::getName)
+                    .sorted()
+                    .collect(Collectors.toList());
+            jobProperties.put("vm", vmNames.isEmpty()? "N/A" : StringUtils.join(vmNames, ", "));
```

Comment:
since Java 8 String#join can be used instead but I'd rather collect the stream to a string (`Collectors.joining(", ")`) and then check if that string is empty

Reply:
Agree, that's it more short & elegant.
Fixed.